arrays/task4.js

```diff
@@ -0,0 +1,6 @@
+const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+
+const evenNumbers = numbers.filter((el) => el %2 === 0 && Number.isInteger(el))
```

Comment:
`&& Number.isInteger(el)` is redundant in this case but if you want to leave it then it should be first check and only then `el %2 === 0`

Reply:
Thanks